### DISCUSSION TITLE
EA Authority and pseudonymous authorization certificates

### DISCUSSION THREAD

**Discussion by Enzar93** [riebl/artery#240] [Q&A]
Hello,

We are a group of students working on a research subject which consists in adding a Security layer via PKI to VANET Network and we'd like to contribute to the Artery project. 
We saw on Vanetza there are EA, AA and CA. However, it seems that Artery doesn't use an EA authority.

If you can help us, we have a few questions : 
- Is the Artery project using an EA authority ?
- Does Artery provide pseudonymous authorization certificates ?

Thanks in advance,
Best regards,

Enzar.

**Comment by riebl** [comment]
No, the simulated networks in Artery do not incorporate any PKI communication at the moment. All network participants are expected to be already fully enrolled and possess their authorization tickets from the simulation's beginning. With security enabled in Artery, the security overhead in terms of increased packet length is considered, but no additional traffic to enrol stations or update any pseudonyms.
